cmd/influxd/server_integration_test.go

```diff
@@ -1462,9 +1462,13 @@ func TestSingleServerDiags(t *testing.T) {
 	config.Monitoring.Enabled = true
 	config.Monitoring.WriteInterval = main.Duration(100 * time.Millisecond)
 	nodes := createCombinedNodeCluster(t, testName, dir, 1, config)
+	createDatabase(t, testName, nodes, "mydb")
```

Comment:
Oh, OK, to repro the error.

Well, let me run your code, and I'll see if I can get answers to my questions below.